datafusion/physical-plan/src/stream.rs

```diff
@@ -241,8 +241,17 @@ impl RecordBatchReceiverStreamBuilder {
         self.inner.spawn(task)
     }
 
-    /// Spawn a blocking task that will be aborted if this builder (or the stream
-    /// built from it) are dropped
+    /// Spawns a blocking task that attempts to send `RecordBatch` data to this stream.
+    ///
+    /// If this builder (or the stream built from it) is dropped **before** the task starts,  
```

Comment:
I think here can be some confusion. Task will never executes does that mean the task ignored or it is stuck in waiting queue forever?

Reply:
I think "task will never execute" means it is droped and never runs.